src/main/java/seedu/address/model/event/Event.java

```diff
@@ -1,5 +1,7 @@
 package seedu.address.model.event;
```

Comment:
If possible can add a method that returns a list of the Persons that is tagged to the event? would make generating statistics aspect much smoother